pipelines/upgrade/07-server_to_final.yaml

```
@@ -20,6 +20,9 @@
       scenario_os: "{{ pipeline_os }}"
       scenario_version: "{{ forklift_upgrade_version_final }}"
     - foreman_server_repositories
+    - role: workaround_foreman_remote_execution_core
+      when:
+        - forklift_upgrade_version_final == "nightly"
```

Should this only be if the OS family is Red Hat?
```suggestion
        - forklift_upgrade_version_final == "nightly"
        - ansible_os_family == "RedHat"
```